script/Arcueid/Day 2/02_00_ARC02_4B.txt

```diff
@@ -251,7 +251,8 @@ Besides, Hisui-chan has also agreed to this. Even if Akiha-sama finds out, I'm s
 [sha:fde32c364c38ee6d2c5efb72dd24436f31f8fa1b]{
 -- Page 214, Offset 3173.
 -- 「そうか、あの子も……」
-"I see, that girl also..."
+//
+"I see, so that girl too..."
```

Comment:
Maybe
> "I see, so she did too..."

Reply:
Would break Kohaku's scolding.

This is one of the harder instance of getting rid of "that girl"

Reply:
Oh, right, I forgor